src/libraries/System.Drawing.Common/src/System/Drawing/BufferedGraphics.Unix.cs

```diff
@@ -66,7 +66,7 @@ public void Dispose()
             if (membmp != null)
             {
                 membmp.Dispose();
-                membmp = null;
+                membmp = null!;
```

Comment:
Github won't let me comment but `Render(Graphics target)` can take a nullable `target`. If the target is null it will be a no-op.